### PR TITLE
Add vGPU policy support

### PR DESCRIPTION
**Pull request by adezxc**
This is a simple PR that adds methods and types to read and manage vGPU profiles.

Also updates the `VdcComputePolicyV2` to support creation and management of vGPU policies.

### CODE REVIEW

**Comment by adezxc** [comment]
Reminder: create changelog entry when we consider the feature complete

File: govcd/vgpu_profile.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+ */
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// VdcComputePolicyV2 defines a VDC Compute Policy, which can be a VM Sizing Policy, a VM Placement Policy or a vGPU Policy.
```
Comment (adambarreiro) [inline govcd/vgpu_profile.go:14]: Probably a wrong description

File: govcd/vgpu_profile.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+type VgpuProfile struct {
+	VgpuProfile *types.VgpuProfile
+	client      *Client
+}
+
+func (client *VCDClient) GetAllVgpuProfiles(queryParameters url.Values) ([]*VgpuProfile, error) {
```
Comment (adambarreiro) [inline govcd/vgpu_profile.go:20]: Missing public methods docs (this one and the ones down below)

**Review by adambarreiro — APPROVED** [review]
Nice, just a few things to change/add

**Review by Didainius — APPROVED** [review]
This one looks simple, but need to get over Terraform :)

**Review by dataclouder — APPROVED** [review]

**Pull request merged**